modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SwiftCombineClientCodegen.java

```diff
@@ -0,0 +1,817 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
I'll remove this in another PR as it'a a new file

https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license